src/DynamoCoreWpf/ViewModels/Core/ShortcutToolbarViewModel.cs

```diff
@@ -9,9 +10,30 @@ public ShortcutToolbarViewModel()
             NotificationsNumber = 0;
         }
 
+        private int notificationsNumber;
+
         /// <summary>
         /// This property represents the number of new notifications 
         /// </summary>
-        public int NotificationsNumber { get; set; }
+        public int NotificationsNumber { 
+            get {
+                return notificationsNumber;
+            } 
+            set {
+                notificationsNumber = value;
+                RaisePropertyChanged(nameof(NotificationsCounterVisibility));
+            }
+        }
+
+        public Visibility NotificationsCounterVisibility { 
```

Comment:
We dont usually store property about `Visibility` but `Bool` and rely on converters, I would rename `NotificationsCounterVisibility` to `IsNotificationsCounterVisibile` and use converter to control it. This will make the ViewModel code less dependent on Windows machine

Reply:
Thanks! Changed.